test/notifier.test.js

```diff
@@ -1057,6 +1057,38 @@ describe("Notifier._buildPayload()", function() {
     done();
   });
 
+  it("should have the most recent frame last", function(done) {
+    var notifier = new Notifier();
+
+    function first() {
+      second();
+    }
+
+    function second() {
+      last();
+    }
+
+    function last() {
+      nonExistant(); 
+    }
+
+    var err;
+    try {
+      first();
+    } catch (e) {
+      err = e
+    }
+    var payload = notifier._buildPayload(new Date(), 'error', 'frame order test', TK(err));
+    expect(payload.data.body.trace.frames).to.be.an('array');
+
+    var numFrames = payload.data.body.trace.frames.length;
+    expect(payload.data.body.trace.frames[numFrames - 3].method).to.equal('first');
+    expect(payload.data.body.trace.frames[numFrames - 2].method).to.equal('second');
+    expect(payload.data.body.trace.frames[numFrames - 1].method).to.equal('last');
```

Comment:
Oh @coryvirok!! I think it's ok.

Our test say `should have the most recent frame last`. So in the context of the error, the **most** recent frame is `last`. Since the frames are:

```javascript
last()
second()
first()
// here all the other frames
```

So I think it's correct.

If we want to release this with webpack and tracekit replacement we can close this an merge https://github.com/rollbar/rollbar.js/pull/82 into `replace-tracekit`.